Document new Roslyn Analyzer Rules for Wilson Token Validation (#25911)
dotnet/docs: docs/fundamentals/code-analysis/quality-rules/security-warnings.md

|[CA5401: Do not use CreateEncryptor with non-default IV](ca5401.md)|Symmetric encryption should always use a non-repeatable initialization vector to prevent dictionary attacks.|
|[CA5402: Use CreateEncryptor with the default IV](ca5402.md)|Symmetric encryption should always use a non-repeatable initialization vector to prevent dictionary attacks.|
|[CA5403: Do not hard-code certificate](ca5403.md)|The `data` or `rawData` parameter of a <xref:System.Security.Cryptography.X509Certificates.X509Certificate> or <xref:System.Security.Cryptography.X509Certificates.X509Certificate2> constructor is hard-coded.|

> | [CA5404: Do not disable token validation checks](ca5404.md) | Setting  <xref:Microsoft.IdentityModel.Tokens.TokenValidationParameters> properties controlling token validation to `false`. |

Reviewer: ```suggestion
| [CA5404: Do not disable token validation checks](ca5404.md) | <xref:Microsoft.IdentityModel.Tokens.TokenValidationParameters> properties that control token validation should not be set to `false`. |
```
